src/securityPolicy.c

```diff
@@ -259,6 +260,9 @@ security_policy_t policyForSignTxInit(
 	WARN_IF(protocolMagic != MAINNET_PROTOCOL_MAGIC);
 
 	// Could be switched to POLICY_ALLOW_WITHOUT_PROMPT to skip initial "new transaction" question
+	if (numCollaterals != 0) {
+		WARN();
+	}
```

Comment:
Instead of this, we should explicitly deal with numCollaterals in each of the cases above (`switch (txSigningMode)`), with DENY or WARN as appropriate.

Reply:
This'll solve itself with the new signing mode I think.